src/System.Text.Primitives/System/Text/Parsing/Unsigned.cs

```diff
@@ -0,0 +1,301 @@
+﻿
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// NOTE: This file is generated via a T4 template. Please do not edit this file directly. Any changes should be made
+// in Unsigned.tt.
+
+namespace System.Text
+{
+    public static partial class PrimitiveParser
+    {
+        public static bool TryParseByte(ReadOnlySpan<byte> text, out byte value, out int bytesConsumed, EncodingData encoding = default(EncodingData), TextFormat format = default(TextFormat))
+        {
```

Comment:
I would throw. We return false if the data passed in is not parable. The format is just not supported (i.e. it will never succeed; no reason to call eh API with unsupported format).